.github/workflows/example-testing.yml

```diff
@@ -0,0 +1,68 @@
+name: Example Tests
+
+on: [push, pull_request]
+
+jobs:
+  examples:
+    runs-on: ubuntu-latest
+    container:
+      image: pgbrodrick/isofit
+
+    steps:
+    - name: Checkout
+      uses: actions/checkout@v2
+
+    - name: Install Isofit
+      run: |
+        pip3 install --editable .
+
+    - name: Download and compile 6Sv
+      run: |
+        cd examples/
+        wget -nv https://github.com/ashiklom/isofit/releases/download/6sv-mirror/6sv-2.1.tar
+        mkdir -p 6sv-2.1
+        tar -xf 6sv-2.1.tar --directory 6sv-2.1
+        cd 6sv-2.1
+        # Necessary for compilation to work with recent GCC versions
+        sed -i Makefile -e 's/FFLAGS.*/& -std=legacy/'
+        make
+        # Pass environment variable to following jobs
+        echo "SIXS_DIR=$(readlink -f .)" >> $GITHUB_ENV
+
+    - name: Download sRTMnet
+      run: |
+        cd examples/
+        mkdir sRTMnet_v100
+        cd sRTMnet_v100
+        wget -nv https://zenodo.org/record/4096627/files/sRTMnet_v100.zip
+        unzip sRTMnet_v100.zip && rm sRTMnet_v100.zip
+        # Pass environment variable to following jobs
+        echo "EMULATOR_PATH=$(readlink -f ./sRTMnet_v100)" >> $GITHUB_ENV
+        # Disables MODTRAN
+        echo "MODTRAN_DIR=\"\"" >> $GITHUB_ENV
+
+    # - name: Running Example - image_cube/small
```

Comment:
@jammont - is this commenting out intentional or was this temporary while you were getting other pieces to work?  Otherwise all looks good!

Reply:
In person: answer is yes, it's intentional due to amount of time on the runs.